src/stores/room-list/previews/MessageEventPreview.ts

```diff
@@ -55,7 +55,11 @@ export class MessageEventPreview implements IPreview {
         }
 
         if (hasHtml) {
-            body = sanitizedHtmlNodeInnerText(body);
+            body = sanitizeHtml(body, {
+                // only allow italics, underlined, code, strikethrough
+                // and bold text in message previews
+                allowedTags: ['em', 'u', 'code', 'del', 'strong'],
```

Comment:
we should also include `i` and `b` in here as the more common options for some other clients.

Reply:
Added 'i' and 'b' tags.